pyproject.toml

```diff
@@ -15,111 +34,20 @@ extend-exclude = '''
-# Poetry:
-#   poetry install --extras dev
-# PIP:
-#   pip install -e .[dev]
-
-[tool.poetry]
-name = "screenpy_requests"
-version = "4.0.3"
-description = "ScreenPy extension to enable interacting with Requests."
-authors = ["Perry Goy <perry.goy@gmail.com>"]
-maintainers = ["Marcel Wilson"]
-license = "MIT"
-repository = "https://github.com/ScreenPyHQ/screenpy_requests"
-documentation = "https://screenpy-requests-docs.readthedocs.io"
-readme = "README.md"
-classifiers = [
-    "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Development Status :: 5 - Production/Stable",
-    "Intended Audience :: Developers",
-    "Topic :: Software Development :: Quality Assurance",
-    "Topic :: Software Development :: Testing",
-    "Topic :: Software Development :: Testing :: BDD",
-    "Natural Language :: English",
-    "License :: OSI Approved :: MIT License",
-]
-
-# It's possible to add optional dependencies with the poetry CLI tool using:
-#   poetry add --optional some_dev_pkg
-# This will set `optional = true` flag in [tool.poetry.dependencies], as seen below
-# But then you need to remember to manually edit the [tool.poetry.extras] dev section
-# and declare the package. This allows `pip install .[dev]` to work as expected
-# Poetry users will need to use the `--extras dev` option rather than the `--with dev`
-# so we dont have two different sets of package versions to update.
-
-[tool.poetry.dependencies]
-python = "^3.8"
-screenpy = ">=4.0.2"
-requests = ">=2.26.0"
-types-requests = ">=2.26.0"
-
-
-# convenience packages for development of screenpy_requests only
-black = {version = "*", optional = true}
-coverage = {version = "*", optional = true}
-mypy = {version = "*", optional = true}
-pre-commit = {version = "*", optional = true}
-pytest = {version = "*", optional = true}
-pytest-mock = {version = "*", optional = true}
-ruff = {version = ">=0.2.0", optional = true}
-sphinx = {version = "*", optional = true}
-tox = {version = "*", optional = true}
-
-[tool.poetry.extras]
-dev = [
-    "pre-commit",
-    "pytest",
-    "pytest-mock",
-    "tox",
-]
-dev_all = [
-    "black",
-    "coverage",
-    "mypy",
-    "pre-commit",
-    "pytest",
-    "pytest-mock",
-    "ruff",
-    "sphinx",
-    "tox",
-]
-test = [
-    "coverage",
-    "pytest",
-    "pytest-mock",
-]
-
-[build-system]
-requires = ["poetry-core>=1.2.0"]
-build-backend = "poetry.core.masonry.api"
-
 
 [tool.ruff]
 target-version = "py38"  # minimum supported version
 line-length = 88  # same as Black.
-
 extend-exclude = [
-    "screenpy_requests/__init__.py",
-    "screenpy_requests/__version__.py",
+    "__init__.py",
```

Comment:
Same thing here about the `__init__.py` i mentioned in ScreenPy: Playwright. I don't think we want to ignore *all* of them, and we can probably just remove this exclusion.

Reply:
Updated.